beacon-light-client/plonky2/circuits_executables/src/utils.rs

```diff
@@ -192,6 +192,19 @@ impl<'a> CommandLineOptionsBuilder<'a> {
     pub fn get_matches(self) -> ArgMatches {
         self.command.get_matches()
     }
+
+    pub fn with_protocol_options(self) -> Self {
```

Comment:
I think this should be called with_balance_verification_options, just in case we add anything else to the set of options